include/boost/config/compiler/diab.hpp

```diff
@@ -12,8 +12,15 @@
 
 #include "boost/config/compiler/common_edg.hpp"
 
-#define BOOST_HAS_LONG_LONG
 #define BOOST_NO_TWO_PHASE_NAME_LOOKUP
+#define BOOST_BUGGY_INTEGRAL_CONSTANT_EXPRESSIONS
+
+#define BOOST_MPL_CFG_NO_HAS_XXX_TEMPLATE
+#define BOOST_LOG_NO_MEMBER_TEMPLATE_FRIENDS
```

Comment:
This should really be changed in Boost.Log.

Reply:
Never mind, I don't see why this should hold things up, will merge.

Reply:
Noted, I'll incorporate it into my pull for Boost.Log, (that admittedly been ongoing for a few weeks,) I'll get back to it soon. - and update this one as well.